archey/logos/darwin.py

```diff
@@ -52,6 +52,20 @@
     """{c[0]}       {c[6]}####{c[0]}     {c[6]}#####{c[0]}    """,
 ]
 
+# Alternative logo : Apple's "retro" logo (low intensity).
+
+COLORS_RETRO_LOW = [
+    Colors.CLEAR,
```

Comment:
```suggestion
    Colors.CYAN_NORMAL,
```

Just before accepting this PR, would you mind cyan (non-bright) as background/entries color (for consistency with regular `retro` style) ? @serpent213

Reply:
Up @serpent213 ! 🙇‍♂️

Reply:
> Just before accepting this PR, would you mind cyan (non-bright) as background/entries color (for consistency with regular `retro` style) ? @serpent213

No big deal. Is cyan special in any way? Clear appears more straight forward to me.

Reply:
Not really, just for consistency. If cyan suits you too, I'd rather go for it 😉
Thanks for the commit ! 🙏